### ISSUE TITLE
Plugin disabled by forge people few days ago

### ISSUE THREAD

**Issue by BSarmady** [anatawa12/ForgeGradle-1.2#72]
Forge guys deleted the following file from their maven, effectively disabling your plugin.

https://files.minecraftforge.net/maven/de/oceanlabs/mcp/versions.json 

I find it irony that every time I want to compile my minecraft mods, I have to downgrade java (environment settings, path ... ), then downgrade gradle too (environment, path), then rely on resources (libraries, simple information) hosted by someone else which I don't even know if they will be there in a year later.

You can see their usual behavior in this forum post, (not supported, closing the thread)
https://forums.minecraftforge.net/topic/100004-451252

Since their plugin doesn't work with gradle 6.x.x, your plugin becomes the only choice to compile minecraft mods. with this file deleted I cannot even verify the behavior of the mod and fix bugs on 

I guess to avoid such time bombs, there should be an option to use files from local folder without needing to refer to an online source. same goes for dependencies and repositories that you are adding to project during runtime.

Thanks in advance

**Comment by anatawa12** [comment]
@bobsort Thank you for reporting!
But I think it's because of the outdated my ForgeGradle fork.
Does this problem still exist with `ForgeGradle 1.2-1.0.5-835562ed`? Testing with clean docker ubuntu environment and [ForgeGradle-example](https://github.com/anatawa12/ForgeGradle-example.git), error about `https://files.minecraftforge.net/maven/de/oceanlabs/mcp/versions.json` are reported on the console. Could you show me the console output which made you know this problem?

**Comment by BSarmady** [comment]
Essentially I build my code for latest, 1.12.2 and 1.47. 

I managed to build my code for 1.4.7 with MCP tools alone but would be great if I code use this again.

Your fork was working until last Tuesday when I last compiled successfully (2.3). On Saturday build started failing on same code base. I looked at console and the error says 404 for version.json file followed by multiple cannot find package in repositories.

As for 2.3 version, I can build again since forge people put the file back again, but I'm not sure when they will decide to remove it again. 

Basically what happened was gradle plugin couldn't find version.json file so it wasn't able to find correct MCP version to download. (I didn't get a screenshot of error) but it was a whole bunch of error messages keep running on log for pages.

**Comment by anatawa12** [comment]
So, The problem is about anatawa12/ForgeGradle-2.3, isn't it? If only about 2.3, Please close this issue and we'll discuss anatawa12/ForgeGradle-2.3#34

**Comment by BSarmady** [comment]
Actually both stopped working and I imagine this one is working again just like the other one, but since I have already setup MCP environment, I'm too lazy to setup gradle again. 

Thank you for suggestion, will close this.

**Comment by BSarmady** [comment]
Actually both stopped working and I imagine this one is working again just like the other one, but since I have already setup MCP environment, I'm too lazy to setup gradle again. 

Thank you for suggestion, will close this.
